lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1860,6 +1860,26 @@ void Aten__Getitem__TOp::getCanonicalizationPatterns(
   });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenIsFloatingPointOp
+//===----------------------------------------------------------------------===//
+
```

Comment:
Curious, would this also work as a ::fold by returning BoolAttr?
This would have the advantage that one doesn't need to explicitly call the canonicalize pass.

Reply:
Yes, maybe implement this as folder is more suitable. I will write it as folder.